tests/tck/features/expressions/typeConversion/TypeConversion2.feature

```diff
@@ -122,6 +122,7 @@ Feature: TypeConversion2 - To Integer
       | 42   |
     And no side effects
 
+  @skip
```

Comment:
can you update this file from https://github.com/opencypher/openCypher/blob/master/tck/features/expressions/typeConversion/TypeConversion2.feature and remove

Reply:
File updated. 
Tags ```@leak``` and ```@NegativeTest``` were removed because are not part of original tck file, but I'm not sure if those should exists in redisgraph project.